scratch/es-to-aoss-migration/dump-aoss-lidvids.py

```diff
@@ -0,0 +1,81 @@
+import json
+import os
+
+import requests
+from opensearchpy import RequestsAWSV4SignerAuth, OpenSearch, RequestsHttpConnection
+from requests_aws4auth import AWS4Auth
+
+from pds.registrysweepers.utils.db import query_registry_db_with_search_after
+from pds.registrysweepers.utils.db.client import get_opensearch_client_from_environment, get_aws_aoss_client_from_ssm, \
+    get_aws_credentials_from_ssm
+
+#START VALIDATED CODE
+iam_role_name = 'temp-mcp-ec2-opensearch-role'
+aoss_host = 'b3rqys09xmx9i19yn64i.us-west-2.aoss.amazonaws.com'
+
+credentials = get_aws_credentials_from_ssm(iam_role_name)
+
+auth = RequestsAWSV4SignerAuth(credentials, 'us-west-2', 'aoss')
+
+# ## VALIDATED
+# url = f'https://{aoss_host_endpoint_url}/_search'
+# response = requests.post(url=url, data="", auth=auth)
+# print(response.content)
+# exit(0)
+
+## VALIDATED
+def get_client() -> OpenSearch:
+    client = OpenSearch(
+            hosts=[{"host": aoss_host, "port": 443}],
+            http_auth=auth,
+            use_ssl=True,
+            verify_certs=True,
+            connection_class=RequestsHttpConnection
+        )
+    return client
+# print(client.search())
+# exit(0)
+
+index_configs = [
+    {'name': 'registry', 'sort_fields': ['lidvid']},
+    {'name': 'registry-refs', 'sort_fields': ['collection_lidvid', 'reference_type', 'batch_id']},
+    {'name': 'registry-dd', 'sort_fields': ['_id']},
+]
+
+node_names = [
+    'img',
+    'rms',
+    'psa',
+    'en',
+    'geo',
+]
+
+def all_docs_query():
+    return {
+    "query": {
+        "match_all": {}
+    }
+}
+
+dump_parent_path = os.path.expanduser('~/serverless-opensearch-dumps')
+os.makedirs(dump_parent_path, exist_ok=True)
+
+for node_name in node_names:
+    for index_config in index_configs:
+        index_type_name = index_config['name']
+        index_name = f'{node_name}-{index_type_name}'
+        sort_fields = index_config['sort_fields']
+
+        output_filename = index_name
+        output_filepath = os.path.join(dump_parent_path, output_filename)
+
+        print(f'Dumping {index_name} to {output_filepath}')
```

Comment:
Should this go to `file=sys.stderr`? (Or in other words, is the output of this program ever going to be used as the input to another?)